packages/shared/dao/sagas.ts

```diff
@@ -66,11 +66,11 @@ function* loadCatalystRealms() {
   if (!PREVIEW) {
     const network: ETHEREUM_NETWORK = yield select(getSelectedNetwork)
 
-    const cachedRealm: Realm | undefined = getFromLocalStorage(getLastRealmCacheKey(network))
+    const cachedRealm: Realm | undefined = yield getFromPersistentStorage(getLastRealmCacheKey(network))
```

Comment:
```suggestion
    const cachedRealm: Realm | undefined = yield call(getFromPersistentStorage, getLastRealmCacheKey(network))
```

To wait for async functions inside saga generators, please use `yield call(fn)` instead of `yield fn()`, it makes the code more testable. More info here: https://redux-saga.js.org/docs/advanced/Testing/, search for ".provide(" to view examples.

Reply:
got it, ty